src/MemberList.cpp

```diff
@@ -29,10 +29,16 @@ MemberList::MemberList(const QString &room_id, QObject *parent)
     } catch (const lmdb::error &e) {
         nhlog::db()->critical("Failed to retrieve members from cache: {}", e.what());
     }
+
+    // HACK: until https://bugreports.qt.io/browse/QTBUG-102169 is fixed, we'll just load all of the
+    // members to make searches work properly. If QTBUG-102169 has been fixed, please delete this
+    // code.
+    while (canFetchMore({}))
+        fetchMore({});
```

Comment:
How about you just pass a larger size to getMembers?

Reply:
Does loading chunks of 500 sound fine?

Reply:
No, just load everything in one call :D

Reply:
Like there is already a call to fetch members a few lines further up, just pass -1 or so to that.